data_collection/web/guides.rst

```diff
@@ -95,6 +100,163 @@ Tracking more sophisticated events might require attaching listeners to the DOM
     </script>
 
 
+
+
+
+E-commerce
+----------
+
+JavaScript API supports 3 types of e-commerce interactions: :ref:`Category and product views<guide_tracking_category_and_product_views>`, :ref:`Cart updates<guide_tracking_cart_updates>` and :ref:`Orders<guide_tracking_orders>`.
+
+.. _guide_tracking_category_and_product_views:
+
+Tracking category and product views
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Usually, the first e-commerce-related action a visitor performs on a website is browsing products. :ref:`setEcommerceView<jtc-api-setEcommerceView>` method allows us to track both category views and product views.
+
+To track a category view, use :ref:`setEcommerceView<jtc-api-setEcommerceView>` method **before** tracking the page view, like this::
+
+    // set category to "Smartphones"
+    _paq.push(["setEcommerceView", false, false, "Smartphones"]);
+
+    // track page view
+    _paq.push(["trackPageView"]);
+
+The same method can be used for tracking product views. Again, it must be called **before** tracking a page view. Example::
+
+    // set product
```

Comment:
Why didn't you use code styling block like `.. code-block:: javascript` ?

Reply:
There are a couple of more code blocks without code styling - if possible please use `.. code-block:: javascript` for them too.

Reply:
This is a simplified syntax for code blocks. You can choose the language globally for a document with `.. highlight:: js` directive. It was missing here though. Thanks for catching that.
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Reply:
Personally I'd prefer to have it literally declared with every code block - maybe that's a little bit more to write but is on the other hand is more clear what is going on.
But that's my personal opinion - you do it as you judgment tells you.